### PR TITLE
add a link to README answering a common question

### PR DESCRIPTION
**Pull request by HackerFoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ that shell. The Vivado wrapper utils/vivado.sh makes sure that the environment
 variables from XRAY_VIVADO_SETTINGS are automatically sourced in a separate
 shell that is then only used to run Vivado to avoid these problems.
 
+**Why 2017.2?** See [#14](https://github.com/SymbiFlow/prjxray/issues/14).
```
Comment (litghost) [inline README.md]: I would make this more explicit, 2017.2 is a requirement, not a suggestion.  This is why.

Comment (mithro) [reply README.md]: Yes, please make it an explicit requirement. Something like;
```markdown
**Why 2017.2?** Currently the fuzzers only work on `2017.2`, see [Issue #14 on prjxray](https://github.com/SymbiFlow/prjxray/issues/14).
**Is 2017.2 really required?** Yes, only `2017.2` works. Until Issue #14 is solved, **only** `2017.2` works and will be supported.
```

Comment (litghost) [reply README.md]: This is still outstanding?

Comment (HackerFoo) [reply README.md]: Done.

**Comment by HackerFoo** [comment]
The format check seems to reject spaces at the end of a line, which is how you force a newline in Markdown.

**Pull request merged**